runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -852,28 +873,28 @@ private void updateTooltip()
 			}
 			else
 			{
-				builder.append("</br>Lower level: ").append(secondsToTime(lowerTime - upperTime));
+				builder.append("</br>Lower level: " + secondsToTime(lowerTime - upperTime));
 			}
 		}
 		else
 		{
-			builder.append("</br>Middle level: ").append(secondsToTime(middleTime - upperTime));
+			builder.append("</br>Middle level: " + secondsToTime(middleTime - upperTime));
 			if (lowerTime == -1)
 			{
 				tooltip = builder.toString();
 				return;
 			}
 			else
 			{
-				builder.append("</br>Lower level: ").append(secondsToTime(lowerTime - middleTime));
+				builder.append("</br>Lower level: " + secondsToTime(lowerTime - middleTime));
 			}
 		}
 		if (raidTime == -1)
 		{
 			tooltip = builder.toString();
 			return;
 		}
-		builder.append("</br>Olm: ").append(secondsToTime(raidTime - lowerTime));
+		builder.append("</br>Olm: " + secondsToTime(raidTime - lowerTime));

```

Comment:
change all this back, "+" basically makes a new stringbuilder, .appends the string behind the other string, and builds. You've already got a stringbuilder so using + instead of .append is a performance loss